kpi/deployment_backends/kc_access/shadow_models.py

```diff
@@ -38,39 +36,6 @@
 )
 
 
-def update_autofield_sequence(model):
```

Comment:
This is still called by `KobocatUser.sync()` in this file, but it's not imported from its new location `kpi/utils/database.py`.

Reply:
OK, this is not actually used anymore:
https://github.com/kobotoolbox/kpi/pull/5037/files/1205888783845f305db12b8cbd9392b1af9f7f41#diff-cbf9041792243e92abae8e95f9af2c66be3e10b1f67292c01bf2d1520d6cf331